agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1742,10 +1742,25 @@ bool backhaul_manager::handle_slave_backhaul_message(std::shared_ptr<SSlaveSocke
             return false;
         }
 
+        //remove this client from other radios
+        for (const auto &slave : m_radio_info_map) {
+            auto associated_clients_map = slave.second.associated_clients_map;
+            for (const auto &vap : associated_clients_map) {
```

Comment:
vap? These are associated clients, right? So `client` would be a better name.

Reply:
Done 👍